src/app/components/Input/index.module.scss

```
@@ -1,8 +1,23 @@
 .input {
-  @apply tw-w-full tw-h-8 tw-p-2 tw-leading-none tw-text-base tw-font-semibold tw-text-black tw-bg-sov-white tw-rounded-lg;
+  @apply tw-w-full tw-h-8 tw-py-2 tw-px-3 tw-leading-none tw-text-base tw-font-semibold tw-text-gray-1 tw-bg-sov-white tw-duration-300 hover:tw-bg-gray-10;
+
+  &::placeholder {
+    @apply tw-text-gray-8;
+  }
+
+  &[value=''], &:not([value]) {
```

I think the input should lighten up when focused by the user.
```suggestion
  &[value='']:not(:focus), &:not([value]):not(:focus) {
```